function_migrations.go

```diff
@@ -340,3 +362,14 @@ type migrateToSpecVersion_previousFunction struct {
 type migrateToBuilderImages_previousFunction struct {
 	Builder string `yaml:"builder"`
 }
+
+// (Defined only for previous versions migration)
+// Invocation defines hints on how to accomplish a function invocation.
+type Invocation struct {
+	Format string `yaml:"format,omitempty"`
+}
+
+// Functions prior to 0.35.0 will have Invocation.Format instead of Invoke
+type migrateToInvokeStructure_previousFunction struct {
+	Invocation Invocation `yaml:"invocation,omitempty"`
```

Comment:
We could rename this struct something like migrateFromInvokeStructure_invocation. If the `yaml:"invocation"` annotation on the struct field stays the same, it should have no effect on the migration.
That way, this struct is no longer part of the package's exported Godocs, and will not get in the way if we ever end up adding the structure back.